packages/ses/src/eval-scope.js

```diff
@@ -1,17 +1,18 @@
-import { FERAL_EVAL, create, freeze, defineProperties } from './commons.js';
+import { FERAL_EVAL, assign, create, freeze, defineProperties } from './commons.js';
 
 export const createEvalScope = () => {
-  const evalScope = create(null);
-  const oneTimeEvalProperties = freeze({
-    eval: {
+  const oneTimeEvalProperties = freeze(assign(create(null), {
```

Comment:
Why not `create(null, {` ?

Reply:
Equivalent but a bit more cumbersome, since there are more descriptor objects. It’s arguably equally reasonable to ask, why not just `{ __proto__: null, ... }`.